senza/cli.py

```diff
@@ -182,13 +197,7 @@ def watching(w: bool, watch: int):
 # and start with an alpha character. Maximum length of the name is 255 characters.
 STACK_NAME_PATTERN = re.compile(r'^[a-zA-Z][a-zA-Z0-9-]*$')
 VERSION_PATTERN = re.compile(r'^[a-zA-Z0-9]+$')
-
-
-def validate_version(ctx, param, value):
-    if not VERSION_PATTERN.match(value):
-        raise click.BadParameter('Version must satisfy regular expression pattern "[a-zA-Z0-9]+"')
-    return value
-
+REGION_PATTERN = re.compile(r'^(eu|ap|us|sa)-(west|east|central|southeast|northeast)-(1|2)$')
```

Comment:
Maybe we could go with a more generic region regex instead of specific list? Maybe `^[a-z]{2}-[a-z]+-[0-9]$` would be enough? Seems like you just want to have a general validation to catch cases of really missing the parameter.